client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -107,14 +156,38 @@ const ProfileInfoEdit = () => {
             <Form.Control
               className="placeholder-text"
               type="email"
+              readonly={"true"}
```

Comment:
If we're not allowing the users to change their email, is that still a good idea to put email as a form here? Personally think we can put it either as a label somewhere in this component or we can remove it from the page.

Reply:
Btw don't forget to merge main into your branch when you're refactoring :)